.appveyor.yml

```diff
@@ -2,7 +2,7 @@ image: Visual Studio 2017
 shallow_clone: true
 
 install:
-  - "set PATH=%PYTHON%;%PYTHON%\\Scripts;%PATH%"
+  - "set PATH=%PYTHON%;%PYTHON%\\Scripts;C:\\cygwin64\\bin;C:\\msys64;%PATH%"
```

Comment:
Is this so the tests won't be skipped because the test which won't find the tool?

Reply:
yes, the qmtest class uses os.environ in where_is to find binaries existence, and for appveyor the cygwin stuff is not in the path by default. This potentially will make other tests not get skipped (which they shouldn't if we want to increase test coverage as much as possible).